src/operators/transformer.operators.ts

```diff
@@ -0,0 +1,26 @@
+import 'reflect-metadata';
+import { Type } from '@nestjs/common';
+
+/* Narrow class-transformer properties by picking the provided fields.
+ */
+export function omitTransformerProperties<T, F extends keyof T>(
+    cls: Type<T>,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _fields: F[],
+): Type<T> {
+
+    // TODO
```

Comment:
I am not attempting to implement `class-transformer` and `class-transformer` operators in this PR. It'll be next.